### PR TITLE
JNG-4088 Add batched support for features.

### PR DESCRIPTION
**Pull request by robertcsakany**
Support added for:

- Batched create for branches
   example: `./project.py -sm judo-meta-psm -tm judo-platform -nf feature/JNG-3834_TestCI "JNG-3834 Test CI 
   capability"`- It creates feature branch, create an empty commit and create draft pull request
- Not only develop branch supported for continous build:
   example: `./project.py -sm judo-meta-psm judo-meta-jql judo-meta-expression -tm judo-platform -bs` - It updates versions and waiting for built. After a new version is builded, update pom.xml, commit and waiting...
- Switch branch
  example: `./project.py -sm judo-meta-psm judo-meta-jql judo-meta-expression -tm judo-platform -sb develop` - switches branches back to develop

### CODE REVIEW

**Review by noherczeg — APPROVED** [review]

File: project.py [hunk 1]
```diff
@@ -67,22 +67,41 @@
...
 
-== Change the dependencies of a given module to the latest versions
-Assuming you've already updated the versions in the project-meta.yml file, you can update one module's dependencies to
-the latest versions:
 
-    ./project.py -ump <module name>
+== Create for branches for projects
 
-You can do the fetching and the updating in one step:
+   ./project.py -sm judo-meta-psm -tm judo-platform -nf feature/JNG-3834_TestCI "JNG-3834 Test CI capability"`
```
Comment (bencegelei) [inline project.py]: backtick at the end

File: project.py [hunk 2]
```diff
@@ -67,22 +67,41 @@
...
 
-    ./project.py -fv -ump <module_name> -gh <github_token>
+   It creates feature branch, create an empty commit and create draft pull request for all projects between judo-meta-psm
+   and judo-platform
+
+== Perform continous build
+
+   ./project.py -sm judo-meta-psm judo-meta-jql judo-meta-expression -tm judo-platform -bs`
```
Comment (bencegelei) [inline project.py]: backtick at the end

File: project.py [hunk 3]
```diff
@@ -67,22 +67,41 @@
...
+
+   It fetches versions for the given modules, updating pom.xml's, pushing and waiting for new versions. It will
+   traverse graph and orchestrating that all descendants have correct and consistent version.
+
+== Switch branch
+
+    ./project.py -sm judo-meta-psm judo-meta-jql judo-meta-expression -tm judo-platform -sb develop`
```
Comment (bencegelei) [inline project.py]: backtick at the end

File: project.py [hunk 4]
```diff
@@ -241,13 +268,16 @@
...
-github_arg_group.add_argument("-cb", "--createfeaturebranch", action="store", dest="create_branch",
+github_arg_group.add_argument("-nf", "--newfeature", action="store", dest="new_feature",
+                              metavar='branch message', nargs=2,
+                              help='Create feature branch and pull request. (same as -cbr and -cpr)')
+github_arg_group.add_argument("-cbr", "--createfeaturebranch", action="store", dest="create_branch",
                               metavar='Feature name', nargs=1,
-                              help='Create feature branch')
-github_arg_group.add_argument("-sb", "--switchfeaturebranch", action="store", dest="switch_branch",
+                              help='Create branch')
+github_arg_group.add_argument("-sbr", "--switchfeaturebranch", action="store", dest="switch_branch",
```
Comment (bencegelei) [inline project.py]: I recommend renaming `--switchfeaturebranch` to `--switchbranch` as well

File: project.py [hunk 5]
```diff
@@ -67,22 +67,41 @@
...
+== Create for branches for projects
 
-You can do the fetching and the updating in one step:
+   ./project.py -sm judo-meta-psm -tm judo-platform -nf feature/JNG-3834_TestCI "JNG-3834 Test CI capability"`
 
-    ./project.py -fv -ump <module_name> -gh <github_token>
+   It creates feature branch, create an empty commit and create draft pull request for all projects between judo-meta-psm
+   and judo-platform
+
+== Perform continous build
```
Comment (bencegelei) [inline project.py]: continous typo

**Review by bencegelei — CHANGES_REQUESTED** [review]
If I get permission to do it, I can fix these while I'm testing the script.

**Review by bencegelei — APPROVED** [review]

**Pull request merged**